### DISCUSSION TITLE
blog/understanding-the-tls-encryption-in-istio/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#326] [Announcements]
# blog/understanding-the-tls-encryption-in-istio/

本文介绍了 TLS、mTLS 相关知识，并介绍了 Istio 中如何开启 mTLS 和其应用场景。

https://jimmysong.io/blog/understanding-the-tls-encryption-in-istio/

**Comment by itbj** [comment]
👍

**Comment by wangyushuai** [comment]
Istio老版本的安全架构是有个NodeAgent的，新的版本直接Pilot-Agent直接链接Citadel了吗， 这样不会有连接数瓶颈嘛

**Reply by rootsongjc** [reply]
你说的“新版本”指的是哪个版本？在 Istio 1.5 版本之后就已经不再有 Citadel 组件了，Istio 的组件回归单体 Istiod了。
